docs/support/troubleshooting.md

```diff
@@ -54,3 +54,14 @@ $ git branch --contains <tag name>
 $ git tag -d <tag name>
 $ git push origin :refs/tags/<tag name>
 ```
+
+## release not found release branch after `git push --force`
+
+**semantic-release** is using [git notes](https://git-scm.com/docs/git-notes) to store information about which releases happened in which branch. Git notes are attached to commits. After git history rewrite due to a `git push --force`, the git notes created by **semantic-release** are lost.
```

Comment:
After git history rewrite => After **a** git history rewrite

Reply:
For more clarity I would suggest:
> **semantic-release** is using both [git tags](https://git-scm.com/docs/git-tag) and [git notes](https://git-scm.com/docs/git-notes) to store information about which releases happened in which branch.
> After a git history rewrite due to a `git push --force`, the git tags and notes associated with commit that were rewritten are lost.

Just to make clear that everything won't be lost, only the things associated with rewritten commits.

Reply:
You can use 

````
```suggestion
Your alternative line here
```
````

to give me a one-click way of applying your suggestion